haiku-apps/textworker/textworker-0.9.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A small utility for text transformations and cleanup"
+DESCRIPTION="TextWorker is a lightweight Haiku-native utility that performs \
+various transformations on text. It supports operations like case conversion, line sorting, trimming, prefix/suffix edits, and more.
+
+This is beta software provided for testing purposes and may contain bugs or incomplete features. Feedback is welcome."
+HOMEPAGE="https://github.com/dospuntos/TextWorker"
+COPYRIGHT="2025 Johan Wagenheim"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="38fea5927508b4fb8a74dedc432fe81d2e769a6a3b030b338bf3b21b0942ffc7"
+SOURCE_DIR="TextWorker-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	textworker$secondaryArchSuffix = $portVersion
+	app:TextWorker = $portVersion
+	"
+REQUIRES="
+	haiku$secundaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	icu74${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+    makefile_engine
```

Comment:
Use 1 tab instead of spaces at the start of the line (just as the others), at least line 37 has the same issue.